desktop/core/base_requirements.txt

```diff
@@ -76,3 +75,4 @@ six==1.16.0
 -e file:///${ROOT}/desktop/core/ext-py3/djangosaml2-0.18.0
 -e file:///${ROOT}/desktop/core/ext-py3/eventlet-0.30.2
 -e file:///${ROOT}/desktop/core/ext-py3/pysaml2-5.0.0
+file:///${ROOT}/desktop/core/ext-py3/thrift-0.16.0
```

Comment:
Curious: Shouldn't it be like line 77?
`-e file:///${ROOT}/desktop/core/ext-py3/thrift-0.16.0`

Reply:
This is the cause the exception we discussed before:

```
  File "/Users/yingchen/github/cloudera/hue/desktop/core/src/desktop/settings.py", line 287, in <module>
    appmanager.load_libs()
  File "/Users/yingchen/github/cloudera/hue/desktop/core/src/desktop/appmanager.py", line 236, in load_libs
    raise Exception("load_apps already has been called.")
```